docs/user/task-manager/task-manager-troubleshooting.asciidoc

```diff
@@ -0,0 +1,699 @@
+[role="xpack"]
+[[task-manager-troubleshooting]]
+== Task Manager Troubleshooting
```

Comment:
It might be helpful to add a table of contents or split this into different sub-pages, as this page is very long and dense (and informative!)